apps/site/lib/site_web/views/schedule_view.ex

```diff
@@ -481,4 +481,26 @@ defmodule SiteWeb.ScheduleView do
   def json_safe_route(route) do
     Route.to_json_safe(route)
   end
+
+  @spec retirement_message(Conn.t()) :: Safe.t()
+  def retirement_message(%Conn{assigns: %{route: %Route{type: 2}}}), do: nil
+
+  def retirement_message(conn) do
+    SiteWeb.ViewHelpers.banner_message(conn, :retirement_message)
+  end
+
```

Comment:
Your code isn't using this anymore so you can delete this!

Reply:
🤦‍♀️